.circleci/config.yml

```diff
@@ -36,6 +36,8 @@ jobs:
           POSTGRES_PASSWORD: easirox
           POSTGRES_USER: postgres
           POSTGRES_HOST_AUTH_METHOD: trust
+          TZ: UTC
```

Comment:
You'll need to set these in SMS before deploying. You can work with @jsclarridge or @rpdelaney to get that done.

Reply:
I had a quick pairing session with @kahlouie. If the TZ/PGTZ environment variables are only needed to configure the postgres container for local and CI testing, then I suspect we don't need to set them in SSM. There's another mechanism for configuring the timezone in the deployed database.